src/components/GoogleAddressLookup/__test__/component.spec.js

```diff
@@ -32,17 +49,34 @@ describe('<PlacesLookupComponent/>', () => {
         });
         expect(global.google.maps.places.AutocompleteService).not.toHaveBeenCalled();
     });
+
     it('should not call google.maps.places.AutocompleteService when isScriptLoaded is true and isScriptLoadSucceed is false', () => {
-        const component = mount(<PlacesLookupComponent className="some-class" />);
+        const component = mount(<PlacesLookupComponent />);
         component.setProps({
             isScriptLoaded: true,
             isScriptLoadSucceed: false,
         });
         expect(global.google.maps.places.AutocompleteService).not.toHaveBeenCalled();
     });
+
     it('should not call AutocompleteService when isScriptLoaded and isScriptLoadSucceed are true but previous isScriptLoaded was true', () => {
-        const component = mount(<PlacesLookupComponent className="some-class" isScriptLoaded />);
+        const component = mount(<PlacesLookupComponent isScriptLoaded />);
         component.setProps(nextProps);
         expect(global.google.maps.places.AutocompleteService).not.toHaveBeenCalled();
     });
+
+    it('should call google.maps.places.AutocompleteService', () => {
+        const component = mount(<PlacesLookupComponent />);
+        component.setProps(nextProps);
+        expect(global.google.maps.places.AutocompleteService).toHaveBeenCalled();
+    });
+
+    it('should be intialized after isScriptLoaded and isScriptLoadSucceed are set to true', () => {
+        const component = mount(<PlacesLookupComponent />);
+        expect(component.instance().fieldRef.current.initialized).toBe(false);
+        component.setProps(nextProps);
+        expect(component.instance().fieldRef.current.initialized).toBe(true);
```

Comment:
here you need to access fieldRef.current because withReduxForm so it is better to export the component itself too in order to test it directly as:
```
export class PlacesLookupComponent extends Component {
     //...
}
export default withReduxForm(PlacesLookupComponent);
```
and the in test import the component as:
```
import { PlacesLookupComponent } from '../component';
```
so the expect should be:
```expect(component.instance().initialized).toBe(true);```